### PR TITLE
165022524 Create endpoint for user login

### PR DESCRIPTION
**Pull request by Fiyiin**
#### Description

This PR creates a login endpoint for the app. The endpoint handles user login 

#### Type of change

Feature: adds the user login method to the User class

#### Pivotal Tracker

#165022524

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,4 +3,6 @@ node_js:
   - "stable"
 services:
   - postgresql
+before_script:
```
Comment (innocentEdosa) [inline .travis.yml]: Hello fiyin
nice work 
I think you should fix a  new line at the end of this file
Thanks

Comment (Fiyiin) [reply .travis.yml]: Fixed. Thanks

**Review by Joyce-O — COMMENTED** [review]
LGTM

**Comment by Bobjayafam** [comment]
LGTM

File: test/test.js [hunk 2]
```diff
@@ -200,3 +200,60 @@ describe('Tests email sender module', () => {
     expect(result.message).to.be.a('string');
   });
 });
+
+describe('Test for user Login', () => {
```
Comment (fuchodeveloper) [inline test/test.js:204]: Kindly add this to a users test file instead of having all the tests in one large file.

**Review by fuchodeveloper — CHANGES_REQUESTED** [review]

**Review by fuchodeveloper — APPROVED** [review]

**Pull request merged**